datafusion/core/tests/sql/expr.rs

```diff
@@ -1629,6 +1630,19 @@ async fn csv_query_sqrt_sqrt() -> Result<()> {
     Ok(())
 }
 
+// this query used to deadlock due to the call udf(udf())
```

Comment:
I don't understand this comment

Reply:
```suggestion
```

Perhaps it was a left over from a copy / paste?

What do you think @izveigor ?